index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
```

Comment:
change the language to zh

Reply:
* Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it❓ 
* Remember to explain why you request a change - it might be obvious for you but not for someone else.